packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/azure/afterappidpinokta.md

```diff
@@ -1 +1,3 @@
+> **Note:** If you want to use a different **Redirect Domain** other than the **Dynamic** default, you can use either **Org URL** or **Custom URL**. See `issuerMode` in the [Identity Provider attributes](/docs/reference/api/idps/#identity-provider-attributes) section.
```

Comment:
How about 
`"If you want to use a specific **Redirect Domain**" instead of the **Dynamic** default....`

Reply:
Thanks @shantanusardal-okta , I'll add this suggestion.